framework/topology.cpp

```diff
@@ -152,49 +153,15 @@ static linux_cpu_info &proc_cpuinfo()
 
 static void reorder_cpus()
 {
-    auto find_cpu_idx_on_same_core_and_packet = [](int cpu_idx) {
-        for (int idx=0; idx < num_cpus(); ++idx)
-            if (cpu_info[idx].cpu_number != cpu_info[cpu_idx].cpu_number &&
-                cpu_info[idx].core_id    == cpu_info[cpu_idx].core_id    &&
-                cpu_info[idx].package_id == cpu_info[cpu_idx].package_id)
-                return idx;
-        return -1;
+    static auto tie = [](const struct cpu_info &cpu) {
```

Comment:
Please name it something like `cpu_tie`.

Reply:
Done.